platform/packages/sdk/src/testing/mod.rs

```diff
@@ -72,8 +78,8 @@ pub fn customized_mock_deps_with_contracts<const N: usize>(
         WasmQuery::CodeInfo { code_id } => SystemResult::Ok(ContractResult::Ok(
             cosmwasm_std::to_json_binary(&CodeInfoResponse::new(
                 *code_id,
```

Comment:
I'm assuming `code_id` is a `Copy` type as it gets dereferenced without then being referenced again, we can drop the reference in favour of having the value itself.
```suggestion
        &WasmQuery::CodeInfo { code_id } => SystemResult::Ok(ContractResult::Ok(
            cosmwasm_std::to_json_binary(&CodeInfoResponse::new(
                code_id,
```